Islandora/Alpaca
# One app to rule them all

**GitHub Issue**: https://github.com/Islandora/documentation/issues/731

# What does this Pull Request do?

Moves Alpaca from an OSGI bundle to a single runnable jar with all the various services inside.

# What's new?

* Removes OSGI bindings
* Pulls SparqlUpdateProcessors in from fcrepo-camel code (it was causing problems with Jena libraries otherwise)
* Create single runnable jar that takes configuration from a properties file.

* Does this change require documentation to be updated? Docs are updated
* Does this change add any new dependencies? No
* Does this change require any other modifications to be made to the repository (i.e. Regeneration activity, etc.)? No
* Could this change impact execution of existing code? Deployment will need to be altered.

# How should this be tested?

Using a playbook or ISLE instance, build the jar and then copy it into the system (somewhere it can access Fedora, Blazegraph, ActiveMQ, the microservices). 

Shutdown Karaf and copy the settings from all the various configuration files to a single properties file. 

Start the jar with that properties file and it should perform all the various actions.

ie.
```shell
java -jar islandora-alpaca-app-2.0.0-all.jar -c config.properties
```

# Additional Notes:

There are some stacktraces thrown that I left in place but are not actually a problem (for instance when you choose a media file, Drupal triggers us to send an event but it doesn't have all the necessary URLs yet. It can be ignored but could also be a sign of a problem.

Could leave the less verbose in the INFO level and move the stacktrace to the DEBUG logging level, but I don't want anyone to miss an actual problem. 🤷 

# Interested parties
@Islandora/8-x-committers (especially those you don't hate Java or at least will do Java, @dannylamb, @elizoller, @adam-vessey, etc)
